tests/AssignProjectFeatureTest.php

```diff
@@ -13,6 +13,7 @@ public function setUp(): void
     {
         parent::setUp();
         $this->cleanupFeatures($this->testFeatureName(), 'project');
+        $this->client->removeUserFeature($this->normalUser['email'], 'can-manage-features');
```

Comment:
potom je v tom teste zbytocne toto https://github.com/keboola/kbc-manage-api-php-client/blob/45577bb8a2e09d0d3a2cb5c59866b85defd24923/tests/AssignProjectFeatureTest.php#L477

Reply:
pravda, to jsem tam zapomněl, je to pryč